src/Game.svelte

```diff
@@ -81,5 +82,7 @@
     <Step>
       <Answer {data} {answer} {question}/>
     </Step>
+  {:else if stepIndex === 8}
+    <Sun colors={['#F7CF52', '#F4581B']} arcAngleRad={Math.PI/2} month="december"/>
```

Comment:
this will be part of the answer component above. and will use `{question.month}` (which is not there yet).